lib/generators/rspec/generators/generator_generator.rb

```diff
@@ -0,0 +1,24 @@
+require 'generators/rspec'
+
+module Rspec
+  module Generators
+    # @private
+    class GeneratorsGenerator < Base
```

Comment:
The class should be `GeneratorGenerator` or the filename should be `generators_generator.rb`, right? In the actual configuration, the generator is invisible to the world.

Reply:
For me it's works only when the filename is `generators_generator.rb`, in this scenario I can use it with `rails g rspec:generators foobar`. Using `generator_generator.rb` and `GeneratorGenerator` for the class doesn't work at all, `rails g rspec:generator foobar` gives `Could not find generator 'rspec:generator'. Maybe you meant "rspec:integration"?`.

I still get `rspec [not found]` when executing `rails g generator foobar`

Reply:
Hmm yes its possible Rails autoloader won't work right with this. Given that 4.0.0 is not yet released we can safely rename this as a breaking change, feel like adding a scenario to test this and then fixing it?

Reply:
@kdiogenes: It should be fixed now on master. https://github.com/rspec/rspec-rails/pull/2217